web/src/Update.elm

```diff
@@ -148,32 +163,61 @@ update msg model =
             List.foldl chain (model ! []) msgs
 
         --     Port Msgs
-        Send sendMsg ->
+        BlockchainSend sendMsg ->
             let
+                -- TODO: model.now does't update in real time.
+                -- TODO: convert model.now from float to ISO format
                 refId =
                     sendMsg.name ++ toString model.now
+
+                mdl =
+                    { model | txReceipts = Dict.insert refId sendMsg model.txReceipts }
+
+                msgs =
+                    [ Debug <| "Blockchain Send : sendMsg = { name = '" ++ sendMsg.name ++ "', payload = '" ++ sendMsg.payload ++ "' }" ]
+            in
+            multiUpdate msgs mdl [ Ports.send ( refId, sendMsg.payload ) ]
```

Comment:
`Ports.send` is ambiguous here. Mentioning as example / reason for changing names soon.